<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetUserGroupWindow</class>
 <widget class="QWidget" name="SetUserGroupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>570</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>用户组设置</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_user">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>用户列表</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QListView" name="user_list">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="new_user">
          <property name="text">
           <string>新建用户</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="edit_user">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>编辑用户</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_user">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>删除用户</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_group">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>用户组列表</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QListView" name="group_list"/>
        </item>
        <item>
         <widget class="QPushButton" name="new_group">
          <property name="text">
           <string>新建用户组</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="edit_group">
          <property name="text">
           <string>编辑用户组</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_group">
          <property name="text">
           <string>删除用户组</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox_info">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>用户/用户组信息</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_id">
            <property name="text">
             <string>用户ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="id_value">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_name">
            <property name="text">
             <string>用户名</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="name_value">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_group">
            <property name="text">
             <string>用户组</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="group_value">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_path">
            <property name="text">
             <string>根目录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="path_value">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_access">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>权限</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QGroupBox" name="file_access">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>文件</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="2" rowspan="2">
            <widget class="QCheckBox" name="file_upload">
             <property name="text">
              <string>上传</string>
             </property>
             <property name="tristate">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0" rowspan="2" colspan="2">
            <widget class="QCheckBox" name="file_delete">
             <property name="text">
              <string>删除</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QCheckBox" name="file_download">
             <property name="text">
              <string>下载</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QCheckBox" name="file_rename">
             <property name="text">
              <string>重命名</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QCheckBox" name="file_no_access">
             <property name="text">
              <string>无权限</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="dir_access">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>目录</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QCheckBox" name="dir_delete">
             <property name="text">
              <string>删除</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="dir_new">
             <property name="text">
              <string>新建</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="dir_rename">
             <property name="text">
              <string>重命名</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="dir_no_access">
             <property name="text">
              <string>无权限</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="edit_access">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>编辑权限</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
